images/win/toolsets/toolset-2016.json

```diff
@@ -75,6 +75,26 @@
             ]
         }
     ],
+    "powershellModules": [
+        {
+            "name": "DockerMsftProvider"
+        },
+        {
+            "name": "MarkdownPS"
+        },
+        {
+            "name": "Pester"
+        },
+        {
+            "name": "PowerShellGet"
+        },
+        {
+            "name": "PSWindowsUpdate"
+        },
+        {
+            "name": "SqlServer"
+        }

```

Comment:
nitpick: may be it makes sense to describe modules without versions in one line?
```
{ "name": "DockerMsftProvider" },
{ "name": "MarkdownPS" },
{ "name": "Pester" },
{ "name": "PowerShellGet" },
{ "name": "PSWindowsUpdate" },
{ "name": "SqlServer" }
```